src/bundle/Resources/config/input_parsers.yml

```diff
@@ -1,101 +1,24 @@
-    ezpublish_rest.input.parser.internal.criterion.ParentLocationRemoteId.class: Ibexa\Rest\Server\Input\Parser\Criterion\ParentLocationRemoteId
-    ezpublish_rest.input.parser.internal.criterion.SectionIdentifier.class: Ibexa\Rest\Server\Input\Parser\Criterion\SectionIdentifier
-    ezpublish_rest.input.parser.internal.criterion.SectionId.class: Ibexa\Rest\Server\Input\Parser\Criterion\SectionId
-    ezpublish_rest.input.parser.internal.criterion.Subtree.class: Ibexa\Rest\Server\Input\Parser\Criterion\Subtree
-    ezpublish_rest.input.parser.internal.criterion.UserMetadata.class: Ibexa\Rest\Server\Input\Parser\Criterion\UserMetadata
-    ezpublish_rest.input.parser.internal.criterion.Visibility.class: Ibexa\Rest\Server\Input\Parser\Criterion\Visibility
-
-    ezpublish_rest.input.parser.internal.sortclause.data_key_value_object.class: Ibexa\Rest\Server\Input\Parser\SortClause\DataKeyValueObjectClass
-    ezpublish_rest.input.parser.internal.sortclause.field.class: Ibexa\Rest\Server\Input\Parser\SortClause\Field
-
-    ezpublish_rest.input.parser.internal.facetbuilder.content_type.class: Ibexa\Rest\Server\Input\Parser\FacetBuilder\ContentTypeParser
-    ezpublish_rest.input.parser.internal.facetbuilder.criterion.class: Ibexa\Rest\Server\Input\Parser\FacetBuilder\CriterionParser
-    ezpublish_rest.input.parser.internal.facetbuilder.date_range.class: Ibexa\Rest\Server\Input\Parser\FacetBuilder\DateRangeParser
-    ezpublish_rest.input.parser.internal.facetbuilder.field.class: Ibexa\Rest\Server\Input\Parser\FacetBuilder\FieldParser
-    ezpublish_rest.input.parser.internal.facetbuilder.field_range.class: Ibexa\Rest\Server\Input\Parser\FacetBuilder\FieldRangeParser
-    ezpublish_rest.input.parser.internal.facetbuilder.location.class: Ibexa\Rest\Server\Input\Parser\FacetBuilder\LocationParser
-    ezpublish_rest.input.parser.internal.facetbuilder.section.class: Ibexa\Rest\Server\Input\Parser\FacetBuilder\SectionParser
-    ezpublish_rest.input.parser.internal.facetbuilder.term.class: Ibexa\Rest\Server\Input\Parser\FacetBuilder\TermParser
-    ezpublish_rest.input.parser.internal.facetbuilder.user.class: Ibexa\Rest\Server\Input\Parser\FacetBuilder\UserParser
-
-    ezpublish_rest.input.parser.internal.route_based_limitation.class: Ibexa\Rest\Server\Input\Parser\Limitation\RouteBasedLimitationParser
-    ezpublish_rest.input.parser.internal.path_string_route_based_limitation.class: Ibexa\Rest\Server\Input\Parser\Limitation\PathStringRouteBasedLimitationParser
 
 services:
     Ibexa\Rest\Server\Common\Parser:
-        class: "%ezpublish_rest.input.parser.class%"
         abstract: true
         calls:
             - [ setRequestParser, [ '@Ibexa\Bundle\Rest\RequestParser\Router' ] ]
 
     Ibexa\Rest\Input\Parser\ContentObjectStates:
         parent: Ibexa\Rest\Server\Common\Parser
-        class: "%ezpublish_rest.input.parser.ContentObjectStates.class%"
+        class: Ibexa\Rest\Input\Parser\ContentObjectStates
```

Comment:
@Nattfarinn looks like the script missed removing entirely `class` from the FQCN service ids here. Merging as-is due to time constraints.